buildSrc/src/main/java/com/google/firebase/gradle/ReleaseGenerator.kt

```diff
@@ -101,25 +141,49 @@ open class ReleaseGenerator : DefaultTask() {
   ) =
     repo
       .log()
-      .addPath("$directory/")
+      .addPath("$directory/CHANGELOG.md")
       .addRange(previousReleaseRef, currentReleaseRef)
       .setMaxCount(1)
       .call()
       .iterator()
       .hasNext()
 
+  private fun getDirChanges(
+    repo: Git,
+    previousReleaseRef: ObjectId,
+    currentReleaseRef: ObjectId,
+    directory: String
+  ) =
+    repo
+      .log()
+      .addPath("$directory/CHANGELOG.md")
+      .addRange(previousReleaseRef, currentReleaseRef)
+      .call()
+      .map { toCommitDiff(repo, it) }
+      .toList()
+
+  private fun toCommitDiff(repo: Git, revCommit: RevCommit): CommitDiff {
+    return CommitDiff(
+      revCommit.id.name,
+      revCommit.authorIdent.name,
+      revCommit.fullMessage.toString(),
```

Comment:
Shouldn't need the `.toString()` here, `RevCommit.fullMessage` is of type `String` already.